### ISSUE TITLE
codewind.dev domain is lost after redirect to eclipse.org landing page

### ISSUE THREAD

**Issue by AndyRWatson** [eclipse/codewind#1235]
**Codewind version:** n/a
**OS:** n/a

**Che version:**.  n/a
**IDE extension version:** n/a
**IDE version:** n/a
**Kubernetes cluster:**.  n/a

**Description of the enhancement:**

when using https://codewind.dev (or https:/codewind.io) the redirect to our landing page on https://www.eclipse.org/codewind also changes the url in my browser.  Could we change the landing page to maintain the originating domain in the url rather than the actual landing page URL (as we used to do with microclimate.dev)

**Proposed solution:**

**Comment by jgwest** [comment]
Presuming our content is still actually hosted on eclipse.org, it would need to host a permanent reverse proxy on whatever server is serving codewind.dev.

**Comment by sghung** [comment]
I don't think we can do anything about this, because the files are hosted on Eclipse. It is similar to Eclipse che being hosted on Eclipse too. I don't think there's anything wrong with us using the Eclipse domain if we're an official Eclipse project. I will put this into icebox for now until someone has time to investigate this further
